jac-uk/admin
# #1865 Scenario test integration

## What's included?
Our QT functionality has moved to the new QT platform. We have already integrated Critical Analysis and Situational Judgement Tests. Here we are integrating Scenario Tests.

- [x] Create a scenario test
- [x] Import responses from the new QT platform
- [x] Input scores
- [x] Process scores into ranked 'zones'
- [x] Incorporate diversity stats
- [x] Set a pass mark
- [x] Override pass/fail status for individual candidates (within a zone)
- [x] Support full screen mode
- [x] Update status of applications based on pass/fail outcome
- [x] Completion: lock things down & display key stats

## Who should test?
✅ Product owner
✅ UTG

## How to test?
For an exercise configured with a Scenario Test please check that you can:
 - See a Scenario Test link in the Tasks side menu
 - Create a Scenario test
 - Transfer participants to the test, including reasonable adjustment data
 - [carry out the test on the new QT Platform]
 - Import responses back from the test
 - Enter scenario test scores:
   - Directly on screen
   - Via copy and paste into/from an external spreadsheet. Please try:
     - Google Sheets
     - Microsoft Excel
     - Your usual spreadsheet editor
 - Enter and exit full screen mode
 - Set an overall pass mark
 - Override individuals outcome
 - Mark as Completed
 - Check that applications status has been correctly updated

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
